testsuite/shared/src/main/resources/org/jboss/as/test/integration/management/cli/README.txt

```diff
@@ -0,0 +1,140 @@
+======= account.keystore =======
+password: elytron
+To read this keystore use following command:
+
+$JAVA_HOME/bin/keytool -list -v -keystore account.keystore -storepass elytron
+
+Keystore type: JKS
+Keystore provider: SUN
+
+Your keystore contains 1 entry
+
+Alias name: account6
+Creation date: Apr 27, 2018
+Entry type: PrivateKeyEntry
+Certificate chain length: 1
+Certificate[1]:
+Owner: CN=account6.key
+Issuer: CN=account6.key
+Serial number: 3c6fb23
+Valid from: Fri Apr 27 20:27:24 CEST 2018 until: Mon Apr 24 20:27:24 CEST 2028
+Certificate fingerprints:
+	 SHA1: DA:1D:B7:63:C8:4B:B1:91:FE:85:60:0C:3F:D0:BB:A1:9E:74:E7:68
+	 SHA256: D0:41:B9:DD:25:53:12:4A:24:BC:ED:07:38:91:45:B7:6F:AA:09:EC:5A:FA:02:B4:32:82:1F:07:B9:F8:0D:76
+Signature algorithm name: SHA256withRSA
+Subject Public Key Algorithm: 2048-bit RSA key
+Version: 3
+
+Extensions:
+
+#1: ObjectId: 2.5.29.14 Criticality=false
+SubjectKeyIdentifier [
+KeyIdentifier [
+0000: 95 64 E7 FB 03 34 E1 17   99 17 9D E8 5A D4 8C 35  .d...4......Z..5
+0010: 04 AB 65 61                                        ..ea
+]
+]
+
+
+
+*******************************************
+*******************************************
+
+
+
+======= test.keystore =======
```

Comment:
I might have missed it but are the entries in test.keystore actually used by any of the tests?